src/lib/ecore/ecore_time.c

```diff
@@ -55,14 +48,16 @@ ecore_time_get(void)
 EAPI double
 ecore_time_unix_get(void)
 {
-//#ifdef HAVE_GETTIMEOFDAY
+#ifdef HAVE_GETTIMEOFDAY
+
    struct timeval timev;
 
    gettimeofday(&timev, NULL);
    return (double)timev.tv_sec + (((double)timev.tv_usec) / 1000000);
-//#else
-//# error "Your platform isn't supported yet"
-//#endif
+#else
+# error "Sua plataforma ainda não é suportada"
```

Comment:
```suggestion
# error "Current platform does not support gettimeofday"
```

Reply:
I was wrong, I solved this problem in the last commit, Tks